DeFiCh/wallet
# `debugging` - add react-devtools

#### What kind of PR is this?:


/kind feature

#### What this PR does / why we need it:
- To allow UI debugging from simulator

#### Which issue(s) does this PR fixes?:

Fixes #761 

#### Additional comments?:
- Tested on iPhone 8 simulator
- Tested on Pixel 3a simulator

Steps to use react-devtools:
- Pre-requisite: `npm ci` to install the dev dependencies (run once)
- Start your desired simulator (only one at a time)
- Start expo that correspond to your simulator, e.g.: `npm run start:android` or `npm run start:ios` 
- In a separate terminal, run `npm run react-devtools:open`, this should open the devtool electron app in a separate window
- On your simulator, trigger developer menu with the shortcut keys, e.g.: ios: `Ctrl-Cmd-Z`, android `Cmd+M`(mac) or `Ctrl+M`(windows)
- Select `Show Element Inspector`
- Tap on any UI, react devtool windows should populate the component hierachy
![image](https://user-images.githubusercontent.com/40191153/133060278-d8173ecd-4293-4c1f-b51c-209d981a36df.png)

#### Developer Checklist:


- [x] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Added translations*
